test/integrations/destinations/criteo_audience/dataDelivery/data.ts

```diff
@@ -1,508 +1,4 @@
-export const data = [
```

Comment:
No need to delete v0 scenarios just create in a separate variable and add in the `data`